code/modules/mob/living/simple_animal/hostile/megafauna/swarmer.dm

```diff
@@ -285,16 +273,6 @@ GLOBAL_LIST_INIT(AISwarmerCapsByType, list(/mob/living/simple_animal/hostile/swa
 	health = 60
 	maxHealth = 60
 	ranged = FALSE
-	login_text_dump = {"
-	<b>You are a swarmer, a weapon of a long dead civilization. Until further orders from your original masters are received, you must continue to consume and replicate.</b>
-	<b>Clicking on any object will try to consume it, either deconstructing it into its components, destroying it, or integrating any materials it has into you if successful.</b>
-	<b>Ctrl-Clicking on a mob will attempt to remove it from the area and place it in a safe environment for storage.</b>
-	<b>Objectives:</b>
-	1. Defend Resource Swarmers while they consume resources and replicate until there are no more resources left.
-	2. Ensure that this location is fit for invasion at a later date; do not perform actions that would render it dangerous or inhospitable.
-	3. Biological resources should not be harmed if possible, violent force can be applied should they fail to keep away.
-	"}
```

Comment:
i didn't even know swarmers had laws? besides i'm pretty sure 95% of people playing swarmer don't actually follow those laws.
even if admins enforced these laws it would be impossible
talking from a gameplay stance here